Magazine homework

daguirreglobant opened the pull request:
Sobreescribo los archivos de mi rama en la tarea 1 con la primera tarea de magazine.

daguirreglobant committed:
magazine homework

daguirreglobant committed:
magazine homework

daguirreglobant committed:
magazine homework

daguirreglobant committed:
magazine homework

emolinaglobant commented:
@daguirreglobant , the title looks misaligned, the font size for the content does not look like the original cover, and looks like the height for the barcode is forced making it look weird

<img width="898" alt="Screen Shot 2022-01-14 at 11 31 34 AM" src="https://user-images.githubusercontent.com/7672155/149550982-12c112df-1428-4d3b-94a1-9ea9576a3df4.png">
.

emolinaglobant commented on daniel-aguirre-salamanca/tarea-1/styles.css:
```diff
@@ -0,0 +1,127 @@
+    text-align: right;
+}
+
+.magazine__content{
```
@daguirreglobant remember to keep alphabetical order in your CSS rules

emolinaglobant commented:
@daguirreglobant congrats on using BEM

daguirreglobant committed:
homework magazine correction

daguirreglobant commented:
@emolinaglobant I reorganized the css alphabetically and changed the background to the same as the original magazine. Added a result of how it looks in my screen as a "result_in_image.png". 

Here you can see a preview: https://raw.githack.com/emolinaglobant/academy-react/daniel-aguirre-salamanca/daniel-aguirre-salamanca/tarea-1/magazine.html

emolinaglobant commented:
@daguirreglobant , nice work!
